backend/inspirehep/records/marshmallow/authors/base.py

```diff
@@ -26,6 +27,17 @@ class Meta:
     email_addresses = NonHiddenRaw(dump_only=True)
 
 
+class AuthorsApiSchema(AuthorsRawSchema):
+    class Meta:
+        exclude = AuthorsPublicSchema.Meta.exclude + ["email_addresses"]
+
+    acquisition_source = fields.Method("get_acquisition_source")
```

Comment:
You could add these in ``AuthorsPublicSchema``

Reply:
The problem is that `AuthorsDetailSchema` is inheriting from `AuthorsPublicSchema` and if I add it I'll exclude emails also from the detailed page and we don't want to do it

Reply:
I see your point, so let's do the following:

We keep it as you did but rename `CollectionApiSchema` to `CollectionPublicListSchema` (i.e. `AuthorPublicListSchema`)